python/mujinwebstackclient/controllerwebclientraw.py

```diff
@@ -213,7 +213,7 @@ def CallGraphAPI(self, query, variables=None, timeout=5.0):
         # repsonse must be 200 OK
         statusCode = response.status_code
         if statusCode != 200:
-            raise ControllerGraphClientException(_('Unexpected server response %d: %s') % (statusCode, raw), statusCode=statusCode, response=response)
```

Comment:
`ControllerGraphClientException` should be removed in this repo

Reply:
Let's do that in a separate step.